app/views/quizzes/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<h1>All Quizzes</h1>
+
+<% if current_user.num_incomplete_quizzes > 0 %>
+  <% num = current_user.num_incomplete_quizzes %>
+  <div class="incomplete-alert">
+    You have <%= num %> incomplete quiz<%= "zes" if num > 1 %>!
```

Comment:
This might be helpful: http://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize